src/main/java/practice/StreamPractice.java

```diff
@@ -50,7 +67,14 @@ public Double getOddNumsAverage(List<Integer> numbers) {
      */
     public List<People> getWorkablePeople(int fromAge, int femaleToAge,
                                           int maleToAge, List<People> peopleList) {
-        return Collections.emptyList();
+        return peopleList.stream()
+                .filter(person -> person.getAge() >= fromAge
+                        && person.getAge() <= maleToAge
+                        && person.getSex().equals(People.Sex.MAN)
+                        || person.getAge() >= fromAge
+                        && person.getAge() <= femaleToAge
+                        && person.getSex().equals(People.Sex.WOMEN))
```

Comment:
```suggestion
                .filter(person -> person.getAge() >= fromAge
                        && (person.getSex() == People.Sex.WOMEN && person.getAge() <= femaleToAge
                        || person.getSex() == People.Sex.MAN && person.getAge() <= maleToAge))
```
Навіщо нам двічі пеервіряти на person.getAge() >= fromAge?

Reply:
Thanks for help. =)